android-pay/src/main/java/com/stripe/wrap/pay/AndroidPayConfiguration.java

```diff
@@ -103,6 +110,37 @@ public static AndroidPayConfiguration init(
         mUsesSources = shouldUseSources;
     }
 
+    /**
+     * Specifies a set of card networks to allow. If not explicitly set,
+     * the default supported networks will be AMEX, DISCOVER, MASTERCARD, and VISA.
+     *
```

Comment:
Maybe add a link to where you found the defaults here?

https://developers.google.com/android/reference/com/google/android/gms/wallet/MaskedWalletRequest.Builder.html#addAllowedCardNetwork(int)

It's a bit messy, but if that changes (totally possible) it'd be nice not to have to rely on the comment.